Update staking-with-realms to Helium Vote

heatedlime opened the pull request:
- Removed references of Realms
- Incorporated instructions for Helium Wallet
- Included IOT and MOBILE staking

heatedlime committed:
Update staking-with-realms.mdx

Removing reference to realms.

Still need to replace images.

waveform06 commented:
staking-with-realms.mdx will be deleted and replaced with staking-with-heliumvote.mdx
Is this PR making that name change assumption?

heatedlime committed:
Update and rename staking-with-realms.mdx to staking-with-realms.md

Updated most images to reflect new process. Please note, I was unable to update the following images:

- Image in "Automatically Created Validator Stakes" as I don't have any locked positions from a validator.

- Landrush image, as my land rush HNT was locked in a ledger, and I don't want to link it to a site to view it.

- Stake Multiplier video was left unchanged, as I'm not sure how to upload a video to GitHub

heatedlime commented:
> staking-with-realms.mdx will be deleted and replaced with staking-with-heliumvote.mdx Is this PR making that name change assumption?

Yes, this is assuming the name is now Staking-with-helium-vote.mdx

heatedlime committed:
Update staking-with-realms.md

Updated images

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -1,82 +1,69 @@
 
-Anyone with HNT in a wallet can create a stake using Realms either using an in-wallet browser or a
-browser-connected wallet.
+Anyone with HNT, MOBILE or IOT in their Helium Wallet App or Solana-compatible browser wallet
```
```suggestion
Anyone with HNT, MOBILE, or IOT in their Helium Wallet App or Solana-compatible browser wallet
```
I'm gonna be that Oxford comma guy.

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -1,82 +1,69 @@
-Anyone with HNT in a wallet can create a stake using Realms either using an in-wallet browser or a
-browser-connected wallet.
+Anyone with HNT, MOBILE or IOT in their Helium Wallet App or Solana-compatible browser wallet
+can be used (such as Phantom orBackpack).
```
sentence ends weird. consider rewriting as 2 sentences

heatedlime replied:
Updated
Anyone with HNT, MOBILE, or IOT in their Helium Wallet App or Solana-compatible browser wallet
(such as Phantom orBackpack) can stake.

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -1,82 +1,69 @@
 
 <figure className="screensnippet-wrapper">
-  <img src={useBaseUrl('/img/realms/connectaccount.png')} />
+  <img src={useBaseUrl('https://github.com/helium/docs/pull/1784#issuecomment-1933058646')} />
```
add images to the /static/img folder in docusaurus

heatedlime replied:
@jthiller I'm a novice when it comes to coding and GitHub. Can you direct me to or provide the link to that folder in docusaurus?

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -1,82 +1,69 @@
 
-Once your wallet has been connected to Realms, your token balances (or existing positions) will be
-displayed in the governance section of the DAO.
+Once in the <code>Governance</code> tab, you will see <code>Your Voting Power</code>. Click <code>Your Voting Power</code>
```
for code formatting, markdown allows using the `  wrapper

jthiller replied:
It is advised to not over-use it as it can impact the reading experience.

heatedlime replied:
Noted. Removed it

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -108,7 +95,7 @@ then it declines linearly until release.
 <figure className="screensnippet-wrapper">
   <img
-    src={useBaseUrl('/img/realms/constantgraph.png')}
+    src={useBaseUrl('/img/s/constantgraph.png')}
```
intent to rename this folder?

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -155,32 +142,41 @@ the overall lockup multiplier.
+### Delegating to a subDAO (veHNT Only)
+
+Once you select how much HNT you want to lock up, an interface will appear to allow you to delegate 
+your veHNT to a subDAO for rewards. A stake will not earn any token return until it is delegated to 
```
```suggestion
your veHNT to a subDAO for rewards. A position will not earn any token return until it is delegated to 
```

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -155,32 +142,41 @@ the overall lockup multiplier.
+
+Once you select how much HNT you want to lock up, an interface will appear to allow you to delegate 
+your veHNT to a subDAO for rewards. A stake will not earn any token return until it is delegated to 
+a subDAO. A stake without delegation earns voting rights, but no token return. You can delegate an 
```
```suggestion
a subnetwork. A stake without delegation earns voting rights, but no token return. You can delegate an 
```

heatedlime committed:
Update docs/governance/staking-with-realms.md

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -155,32 +142,41 @@ the overall lockup multiplier.
 ## Managing Stakes
 
-With Realms, it's a sizzling experience to take control of your HNT stakes, ensuring they're grilled
+With Helium Vote, it's a sizzling experience to take control of your HNT stakes, ensuring they're grilled
```
lol don't feel the need to entertain my silliness here.

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -155,32 +142,41 @@ the overall lockup multiplier.
   </figcaption>
 </figure>
 
 ## Managing Stakes
```
We should note that any changes to delegations will forego the delegation rewards for that day. e.g. moving from mobile to iot

heatedlime replied:
This is already mentioned in the Undelegating section

_Once a stake has been delegated, it can be undelegated in the same interface. Delegations can be
changed at any time, however, it will take one epoch for the position to be available for
redelegation. The stake must be delegated for the entire epoch to receive rewards for that epoch.
Received rewards need to be claimed manually. Only then will they be sent to the wallet the veHNT
position is in. A delegated position can only be undelegated after all accrued rewards have been
claimed._

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -215,13 +213,13 @@ greater or equal duration. This functionality is useful for consolidating positi
-positions after the split is complete.
+You must undelegate both positions before transferring a stake between positions (veHNT only). Once you've 
+undelegated your stake, you can click on your locked position to delegate your stake to a new
+subDAO.
```
Prefer 'subnetwork' language

heatedlime replied:
changed all instances from subdao to subnetwork

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -215,13 +213,13 @@ greater or equal duration. This functionality is useful for consolidating positi
-  <img src={useBaseUrl('/img/realms/transfer.png')} style={{ maxHeight: 300 }} />
-  <figcaption>A position can be transferred in full or in part.</figcaption>
+  <img src={useBaseUrl('https://github.com/helium/docs/pull/1784#issuecomment-1933080044')} style={{ maxHeight: 300 }} />
+  <figcaption>Once your stake is undelegated, you can click on the stake to transfer it.</figcaption>
```
keep 'position' language

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -298,13 +298,13 @@ address for HNT.
+  <figcaption>Transaction warning noting the user does not have enough $SOL in order to continue.</figcaption>
 </figure>
 
-<!-- Assets templates available at https://www.figma.com/file/fZ4r0rVdhSfBrR9qTpT4Qg/Realms-Documentation -->
```
probably worth keeping this

heatedlime committed:
Update staking-with-realms.md

made edits joey recommended

heatedlime committed:
Update docs/governance/staking-with-realms.md

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

heatedlime committed:
Update docs/governance/staking-with-realms.md

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

heatedlime committed:
Update staking-with-realms.md

edits per joey's comments

jthiller committed:
Adds reference image.

heatedlime committed:
added images

heatedlime commented:
@jthiller are you able to check this to see if I did the images right? I'm not sure how to review it and have the images show properly.

jthiller commented:
It looks like some links are broken.

You can check out a staging site by clicking the 'deploy preview' link once it's finished building. 
<kbd><img width="813" alt="Screenshot 2024-02-19 at 2 57 27 PM" src="https://github.com/helium/docs/assets/1965053/c2a2c457-c890-42f7-84c0-c30bcde74cf6"></kbd>

If you take off the `/static/` they should work.
e.g.  
```mdx
<img src={useBaseUrl('img/modular-governance/Your-voting-power.png')} />
```

heatedlime committed:
Update staking-with-realms.md

Edit to try to make images work

heatedlime committed:
Update staking-with-realms.md

small fix

heatedlime committed:
Update staking-with-realms.md

heatedlime committed:
Update staking-with-realms.md

heatedlime committed:
Update staking-with-realms.md

heatedlime committed:
edits

heatedlime committed:
edit

heatedlime commented:
@jthiller sorry for tagging you again. I was almost done with it, but now it appears it's broken somewhere, and I don't know how to check what I broke. It was going smoothly until I tried to re-add the images that weren't uploading..

jthiller commented on docs/governance/staking-with-realms.md:
```diff
@@ -1,84 +1,87 @@
+  style={{ maxHeight: 300, borderRadius: '40px', padding: '14px 16px 20px' }}
+  /> 
+   The Governance tab is the fourth tab from the left. The HNT stake button at the top is the default selection.
   </figcaption>
```
```suggestion
  <img src={useBaseUrl('/img/modular-governance/Governance-tab-on-HW.jpg')}
  style={{ maxHeight: 300, borderRadius: '40px', padding: '14px 16px 20px' }}
  /> 
  <figcaption>
      The Governance tab is the fourth tab from the left. The HNT stake button at the top is the default selection.
  </figcaption>
```

jthiller replied:
@heatedlime Looks like a missing tag. This should fix the broken build.

heatedlime committed:
Update docs/governance/staking-with-realms.md

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

heatedlime committed:
Update staking-with-realms.md

heatedlime committed:
edit

heatedlime committed:
.

heatedlime committed:
Update staking-with-realms.md

heatedlime committed:
This should hopefully be the last commit...

heatedlime committed:
Final Commit

heatedlime committed:
Image edits

heatedlime committed:
size changes

heatedlime committed:
edit constant and decay pics

heatedlime commented:
After 27 commits I finally got it! should be all good for your review @jthiller

jthiller committed:
Linting, standardizing styles.

jthiller committed:
Handle p wrappers on figcaption

jthiller committed:
Remove split-out figcaptions

jthiller committed:
Change staking-with-realms to staking-with-helium-vote

jthiller committed:
Update heliumvote lockup links

jthiller committed:
Content editing

jthiller reviewed (approved):
Thanks for taking this on!
